### PR TITLE
Issue 7948

### PR DESCRIPTION
**Pull request by YuexinChen96**
### Describe your changes :

I worked on adding a Loader when fetching the data in ElasticSearch because it costs some time when fetching data with empty data displayed.

#
### Type of change :

- [] Bug fix
- [x] Improvement
- [] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Comment by harshach** [comment]
@chirag-madlani review please

**Comment by YuexinChen96** [comment]
@chirag-madlani Hi Chirag, I added the Loader to the project and reviewed the failing tests. It seems like it is not caused by my change. Please check, or I will close this pull request.

File: openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/elastic-search-index.page.tsx [hunk 1]
```diff
@@ -155,6 +156,7 @@ const ElasticSearchIndexPage = () => {
             loading={batchLoading}
             size="small"
             title="ElasticSearch">
+            {streamLoading ? <Loader size="small" type="white" /> : ''}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/elastic-search-index.page.tsx:159]:
```suggestion
            {batchLoading ? <Loader size="small" type="white" /> : ''}
```

**Review by chirag-madlani — CHANGES_REQUESTED** [review]
Please rebase your branch with latest main to fix failing checks.
Sorry for the delay in response

**Comment by YuexinChen96** [comment]
I have tried to merge to the main branch and create a new pull request.

**Pull request closed without merging**